### ISSUE TITLE
Wikipedia is not the best comparison for your project

### ISSUE THREAD

**Issue by strypey** [bigokro/gruff#1]
Have you seen the talk pages on Wikipedia? Any good quality debate on MediaWiki sites happens despite the platform, not because of it. I think you would be better to use 'The Git of debates' as your tag line, as that's a better metaphor for what your platform does; breaking debates down into discrete modules that people can contribute to independently of the rest of the work.

**Comment by bigokro** [comment]
I get your point, but there are a number of reasons for the comparison:
- One place for the knowledge to accumulate
- Crowd-sourced
- Not owned by a corporate entity
- Uses moderators (actually, curators)
- Is where you go to get information
- Easy to use and read (no learning curve for the average citizen)

Not to mention that the average person will have heard of Wikipedia, as opposed to Git. The reference is meant to make sense to non-developers. As for the quality of discussion on Wikipedia, that's the reason one is needed specifically for debates. Otherwise, it would just be Wikipedia.

As for your suggestion, I absolutely agree that the Git model matches a lot better the structure of what this project is about.

Btw, are you interested in lending a hand? Right now there's a working group of about 10-15 people. We're working on similar (duplicate) projects in parallel while finalizing the model in a white paper. Then we'll start building out the more definitive version (and git-like forking has been in the discussion).

**Comment by strypey** [comment]
OK, shat you're saying is something like "Gruff is to debates what Wikipedia is to encyclopedias". That makes sense.

Keep me in the loop, for sure. As a consequence of my experiences trying to do collaborative policy development with the Internet and Pirate parties (using Loomio), I've been doing some [general brainstorming](https://www.coactivate.org/projects/disintermedia/gitocracy) on how to create a GitHub style policy development platform ("PolicyHub is to policy development what GitHub is to software development"). As part of that I imagined creating a re-usable, AGPL-licensed, policy editing front-end (codenamed Gitocracy) that uses Git as a back-end. I've also got a project called [CounterClaim](https://www.coactivate.org/projects/drillingfortruth/) (currently just using an oldskool wiki-and-blog), which I imagine being a component of PolicyHub; you can't do effective evidence-based policy without a way of organizing evidence, and that's what CounterClaim is for.

My overall goals with these projects are definitely in alignment with those of Gruff and Democracy Earth, either of these projects could either fit in with the DE toolset, or be implemented by Gruff or something else that DE is already working on. Keen to learn more.

**Comment by bigokro** [comment]
That's it, perfect distillation.

It sounds like you are definitely after the same things we are. If you're interested, please join our Slack team. No obligations there, but you can participate in the conversation or contribute to the white paper as you like, and talk about the work you're doing.

https://join.slack.com/t/gruffdebates/shared_invite/enQtMzEzOTU3NzYyMDY3LTI4YzUxM2I0MjFjZDNlMzQxZDM4YTgwNDNlMTY3YWQwNjJhYjk0ODE1MGU5NzQ2MTAyNTFhZWRhMGNjMjAxNmE

**Comment by strypey** [comment]
@bigokro
> It sounds like you are definitely after the same things we are. If you're interested, please join our Slack team.

Sorry, I'm not a [Slack used](https://opkode.com/blog/slacks-bait-and-switch/), for the same reason I'm not a [FarceBook used](https://stallman.org/facebook.html) :) Let me know if you switch to a free code replacement like MatterMost, RocketChat, Matrix, XMPP, or Loomio though, and I'd be super keen. In
the meantime, feel free to send some reps from your herd to drink at the [Open App Ecosystem](https://www.loomio.org/d/Yxz6uLfV/) waterhole.

**Comment by bigokro** [comment]
FWIW, I'm well aware of their business model, so for me, I wouldn't consider it a bait-and-switch. We're discussing whether or not we want to continue there. I'll let you know if we change our mind on it, but it's working well for us right now. We may sign up for their non-profit plan to get unlimited history, but even that's not so important to me.

If you'd like to make a pitch for one of these other platforms, now would actually be the right time, since we're in the middle of discussions. Maybe drop on in just to introduce yourself and propose an alternative.

**Comment by strypey** [comment]
When commons stewardship organisations like CreativeCommons (and open source projects) started to use it in place of IRC, it was justified on the basis that people didn't have to use the proprietary (and [awful](https://sircmpwn.github.io/2016/11/24/Electron-considered-harmful.html)) Slack client. Instead they could use their choice of IRC or XMPP client. That allowed Slack to enclose organisations that otherwise never have gone there, and now they're locked in, because Slack has their chat history. That's why it's a bait-and-switch.

I honestly don't want to create an Slack account. Is it ok if I pitch a replacement in a GitHub Issue instead? Or on a mailing list? Or a Loomio group? Or a Discourse forum?

A few questions that would help me decide what to recommend:
* are you in a position to self-host a team chat package, or are you only interested in using hosted services?
* do you use Slack mainly for realtime chat, or asynchronous messages, or equally for both?
* is voice and video mission critical, or do you only need text chat?
* how important is your chat history? Is it an essential archive of your institutional memory, like mailing list archives? Is it ephemeral, like idling in IRC, with your team usually summarizing Slack chat in docs on a wiki or whatever?
* is email integration of interest, or are you all drowning in email and not wanting email delivery/ replies as a option? (I'm guessing it's not that important since Slack has no email bridge, but...)

**Comment by bigokro** [comment]
To summarize:
* Async
* No email
* No voice/video (we use Google Hangouts)
* History is debatable. Some think it's essential, some (like me) care more about the results - yes, we summarize in the white paper and other docs
* Unlimited users, although I think the non-profit plan for Slack of 250 should be enough
* Don't want to host

Things I especially like about the Slack client:
* Very readable
* I use Slack for work and other groups, so it's easy to switch back and forth
* Desktop notifications of new messages (NOT email, PLEASE)
* Multiple channels, for more focused discussions of many sub-projects
* The ability to tag posts. We use that for "permalinks", welcome info, and keeping a running list of topics for the next hangout.
* Permalink invite for new members
* Easily searchable (for the history that is available, at least)
* Easy to attach/embed docs and images
* Quick polls for simple voting
* @user mentions

In summary... extremely usable and productive.
